picknik_twist_controller/controller_plugins.xml

```diff
@@ -0,0 +1,7 @@
+<library path="picknik_twist_controler">
+  <class name="picknik_twist_controler/PicknikTwistControler" type="picknik_twist_controler::PicknikTwistControler" base_class_type="controller_interface::ControllerInterface">
+  <description>
+    6dof twist controller
```

Comment:
Can we update this to say it actually does? Subscribes to a twist message and forwards it to the hardware.